src/OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -12,17 +12,28 @@
 #include <time.h>
 #endif
 
+namespace orbit_base {
+
 #ifdef _WIN32
-[[nodiscard]] inline uint64_t MonotonicTimestampNs() {
+
+[[nodiscard]] inline uint64_t TimestampNs() {
   auto time_since_epoch = std::chrono::steady_clock::now().time_since_epoch();
   return std::chrono::duration_cast<std::chrono::nanoseconds>(time_since_epoch).count();
 }
+
 #else
-[[nodiscard]] inline uint64_t MonotonicTimestampNs() {
+
+// Linux clock associated with Orbit's main time domain.
```

Comment:
I would add a comment that this needs to be a monotonic clock.

Reply:
Added in the comment above. Thanks for pointing out.